sound/soc/sof/ipc4-topology.c

```diff
@@ -2947,15 +2946,17 @@ static int sof_ipc4_route_setup(struct snd_sof_dev *sdev, struct snd_sof_route *
 	sroute->src_queue_id = sof_ipc4_get_queue_id(src_widget, sink_widget,
 						     SOF_PIN_TYPE_OUTPUT);
 	if (sroute->src_queue_id < 0) {
-		dev_err(sdev->dev, "failed to get queue ID for source widget: %s\n",
+		dev_err(sdev->dev,
+			"failed to get output queue ID for source widget: %s\n",
```

Comment:
I would use the same terminology as the code

"failed to get src_queue_id ID for source widget:" 

our source queue ID as use line 2971

Reply:
right, but it is the output pin/queue ID of the source widget.
In sroute we store it as the src_queue_id, yes, but when you say: "failed to get src_queue_id ID for source widget:" I would read it that it is the source queue ID of the source widget that failed, while it is not, it is the sink_queue_id of the source widget, which is a source queue in the connection between the two modules.

just overlooking the fact that we use `queue` and `pin` for the same thing, depending on where we are in code..